examples/pt2/README.md

```diff
@@ -0,0 +1,49 @@
+## PyTorch 2.x integration
+
+PyTorch 2.0 brings more compiler options to PyTorch, for you that should mean better perf either in the form of lower latency or lower memory consumption. Integrating PyTorch 2.0 is fairly trivial but for now the support will be experimental until the PyTorch 1.14 release.
```

Comment:
Maybe: ...until the official release. 
As there will technically be no 1.14 release?

Reply:
Yeah lemme make that clearer